### ISSUE TITLE
Regex matcher broken

### ISSUE THREAD

**Issue by karlism** [camptocamp/grafana-prometheus-alertmanager-datasource#100]
Hello,

After an update to 1.0.0, regex matcher seems to be broken, for example following filter expression:
`tenant=~"$tenant",environment="$environment"`
works properly with 1 tenant selected but returns error when multiple tenants are selected.

Plugin creates request to following address:
`api/datasources/proxy/2/api/v2/alerts?active=true&silenced=false&inhibited=false&filter=tenant=~"{network&filter=infra}"&filter=environment="production"`
which returns `Bad request` error with following message: `message:"bad matcher format: tenant=~"{network"`:
```
status:400
statusText:"Bad Request"
data:Object
error:"Bad Request"
response:"bad matcher format: tenant=~"{network"
message:"bad matcher format: tenant=~"{network"
config:Object
url:"api/datasources/proxy/2/api/v2/alerts?active=true&silenced=false&inhibited=false&filter=tenant=~"{network&filter=infra}"&filter=environment="production""
method:"GET"
withCredentials:false
headers:Object
retry:0
hideFromInspector:false
message:"bad matcher format: tenant=~"{network"
```

Grafana version: 7.3.10
Plugin version: 1.0.0

**Comment by karlism** [comment]
Might be related to https://github.com/camptocamp/grafana-prometheus-alertmanager-datasource/issues/81

**Comment by hbollon** [comment]
Hi @karlism,
This issue should be fixed with new [v1.1.0](https://github.com/camptocamp/grafana-prometheus-alertmanager-datasource/releases/tag/v1.1.0) release.
Can you give it a try? I'm closing this issue but if this problem persist don't hesitate to reopen it :+1:

**Comment by karlism** [comment]
@hbollon, thanks, I'll give it a try later this week.
